packages/wallets/src/features/accounts/screens/PersonalDetails/PersonalDetails.tsx

```diff
@@ -1,89 +1,100 @@
 import React, { ReactNode, useEffect } from 'react';
 import { useResidenceList, useSettings } from '@deriv/api';
-import { FlowTextField, useFlow, WalletDropdown, WalletText } from '../../../../components';
+import { FlowTextField, Loader, useFlow, WalletDropdown, WalletText } from '../../../../components';
 import { accountOpeningReasonList } from './constants';
 import './PersonalDetails.scss';
 
 const PersonalDetails = () => {
-    const { data: residenceList } = useResidenceList();
+    const { data: residenceList, isLoading, isSuccess: isResidenceListSuccess } = useResidenceList();
     const { formValues, setFormValues } = useFlow();
     const { data: getSettings } = useSettings();
 
     useEffect(() => {
-        if (getSettings) {
+        if (getSettings && isResidenceListSuccess) {
```

Comment:
There is a bug where if POI is pending/verified and we show POA screen initially, the default values are not filled in for citizenship. This is because of the `useResidenceList` hook still fetching the data so we need to wait for it